specification/src/chapter2.adoc

```diff
@@ -329,6 +380,7 @@ a| * Glitching to bypass secure boot
 * Retrieving residual confidential memory after a system reset
 a| * Implement robust power management
 * Implement cryptographic memory protection with at least boot freshness
+* Code hardening (for example, see https://www.riscure.com/publication/practical-steps-evaluate-protect-secure-boot-implementations-embedded-devices/[Riscure white paper])
```

Comment:
May want to move references to bibliography. This reference is not in public domain - should we replace with a public domain reference?

Reply:
It is public domain, they just ask for an email address. It is a commonly used reference for hardened boot, at least in IOT/edge use cases and many mobile clients, as Riscure also support several certification programmes.

If it is controversial then I can remove the reference. I just thought it easier to just link to that than repeat a lot of low-level technical detail. Happy to take suggestions for other references as well.